kubo-rpc/kubo-rpc.yaml

```diff
@@ -199,6 +199,12 @@ paths:
           schema:
             type: string
           required: false
+        - name: offline
```

Comment:
I would have called this `local` offline feels weird for a http api.

Reply:
Are we cloning a kubo undocumented name?

Reply:
Same, however we are forced to use the same name as Kubo. Good news is we will eventually deprecate and replace all of these API calls with equivalent and correct abstractions in the Ceramic API.